utilities/decrypt_data/decryption_for_program_service.py

```
@@ -0,0 +1,134 @@
+
+def run_migration():
+    table = CONFIG["table"]
+    select_sql = f"""
+      SELECT id,
+             location_code
+        FROM {table}
+       WHERE parent_id IS NULL
+    """
+
+    with psycopg2.connect(CONFIG["db"]["dsn"]) as conn:
+        with conn.cursor(cursor_factory=DictCursor) as cur:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)